packages/xo-server-perf-alert/src/index.js

```diff
@@ -27,7 +27,7 @@ const VM_FUNCTIONS = {
       const filteredLegends = legend.filter(l => l.name.match(regex))
       const accumulator = Object.assign(...filteredLegends.map(l => ({ [l.name]: [] })))
       const getDisplayableValue = () => {
-        const means = Object.keys(accumulator).map(l => mean(accumulator[l]))
+        const means = Object.keys(accumulator).map(l => mean(accumulator[l].map(parseFloat)))
```

Comment:
Avoid `parseInt` and `parseFloat` which can have surprising behaviors and _never_ use `parseInt` with `map`:

- strings starting with `0x` or `0X` will be interpreted as hexadecimal
- when a non numeric character is encounter, it stops parsing and silently drop the rest of the string (e.g. `parseInt('1e2')` → `1`)
- when a second parameter is provided (which is the case for `map`), it used as the radix (e.g. `parseInt('11', 4)` → `5`)

You can use the prefix operator `+` or `Number` if you need a function.

```suggestion
        const means = Object.keys(accumulator).map(l => mean(accumulator[l].map(Number)))
```